pyiron_workflow/nodes/composite.py

```diff
@@ -423,6 +423,11 @@ def color(self) -> str:
         """For drawing the graph"""
         return SeabornColors.brown
 
+    @property
+    def gui_color(self) -> str:
+        """For nodes in the gui"""
+        return "#eacf9f"
```

Comment:
I find it nice that @liamhuber so far mostly followed the convention of naming the color and return the variable. Do you maybe want to do that as well?

Reply:
I added variables for the colors, but did not define them in the Seaborn class (which is in pyiron_snippets). I think the Seaborn colors follow some pre-defined color palette . Is that OK?

Reply:
And where would you like the example picture?

Reply:
In the PR comment. Actually you might need to explain either in the code (better) or in the PR how this property is used.